examples/bank/bank.go

```diff
@@ -195,23 +194,24 @@ func (bank *Bank) initBankAccounts(cash int64) int64 {
 			id := bank.makeAccountID(i)
 			if !accts[string(id)] {
 				batch.Put(id, value)
-				totalCash += cash
+				newCash += cash
 			}
 		}
 		return txn.Run(batch)
 	}); err != nil {
 		log.Fatal(err)
 	}
-	log.Info("done initializing all accounts")
-	return totalCash
+	log.Infof("done initializing all accounts, bank is worth %d$", oldCash+newCash)
+	return oldCash + newCash
 }
 
 func (bank *Bank) periodicallyCheckBalances(expTotalCash int64) {
 	var lastNumTransfers int32
 	lastNow := time.Now()
+	t := time.NewTicker(time.Second)
```

Comment:
you want output every one second, not every `one plus how long the scan takes for that particular iteration`. The output is very jittery. It still is after this change though, because the scan competes with all the transactions on the whole range and needs a lot of restarts.